docs/import-assistant.md

```diff
@@ -0,0 +1,67 @@
+ # Import Assistant
+ 
+Run powerful AI-enabled commands to assist with your import script development.
+
+&nbsp;
+### Install
+Add an npm script entry to your Edge Delivery project's `package.json`:
+
+```
+"assistant": "aem-import-helper assistant"
+```
+
+### Options
+
+- `--url`: The URL of the page to analyze
+- `--outputPath`: The directory to save the import scripts
+- `--name`: The name of the block
```

Comment:
Could we add an **Options** section to each command that includes any additional options that command needs over and aobve the common options?